infra/ansible/inventories/group_vars/all.yaml

```diff
@@ -0,0 +1,10 @@
+ansible_user: ubuntu
+
+# geerlingguy.docker
+docker_users:
+  - ubuntu
+
+filebeat:
+  version: 6.x
+  cloud_id: "{{ lookup('onepassword', 'Filebeat configuration', field='filebeat_cloud_id', vault='Test Infra') }}"
```

Comment:
oh nice! I did not know you could `lookup('onepassword'`

Reply:
I'm not mad about it, since you have to install `op` and get a session token, but at least everyone on the team gets access to 1Password during onboarding, so at least no need to share additional tokens. Keybase encrypted file system is a nice solution as well, but I don't want to push another tool onto the team :)

Reply:
i hear you. those are both neat tricks.